packages/astro/tsconfig.json

```diff
@@ -24,5 +24,11 @@
     "declarationDir": "dist/types",
     "noUncheckedIndexedAccess": true
   },
-  "exclude": ["dist", "build", "node_modules", "src/astro-components"]
+  "exclude": [
+    "dist",
+    "build",
+    "node_modules",
+    "src/astro-components/index.ts",
+    "src/astro-components/control/BaseClerkControlComponent.ts"
```

Comment:
Instead of excluding the whole `src/astro-components` folder, we only exclude the `ts` files for now so that virtual modules can be properly typed